connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -111,13 +114,13 @@ protected static ConfigDef baseConfigDef() {
                 .define(CLUSTER_CONFIG, Type.STRING, CLUSTER_DEFAULT, Importance.HIGH, CLUSTER_CONFIG_DOC)
                 .define(BOOTSTRAP_SERVERS_CONFIG, Type.LIST, BOOTSTRAP_SERVERS_DEFAULT,
                         Importance.HIGH, BOOTSTRAP_SERVERS_DOC)
-                .define(KEY_CONVERTER_CLASS_CONFIG, Type.CLASS,
+                .define(KEY_CONVERTER_CLASS_CONFIG, Type.CLASS, CONVERTER_CLASS_CONFIG_DEFAULT,
```

Comment:
These changes don't seem relevant to the JIRA?

I'm also pretty sure we don't want these as defaults -- the JSON converter is included, but no reason it should be the default as far as I can tell. There's a stronger argument for providing a default for the internal one, but that still seems unrelated to the issue the JIRA is supposed to address.